### DISCUSSION TITLE
Any plan to make a shortcut key to change the brightness? (Not by percent but directly to it)

### DISCUSSION THREAD

**Discussion by ADog55B** [xanderfrangos/twinkle-tray#389] [Ideas]
I've always had the habit to play games at 80 brightness, working at 40 at day, and browsing webs at 20 at night. I'm wondering if a shortcut setting will help. 
![Clip_20220714_151859](https://user-images.githubusercontent.com/27490057/179098061-61298875-7bc8-4bd1-ac36-93ab25d613e6.png)
Here's a pic as a prototype.
Thanks!!

**Comment by xanderfrangos** [comment]
Yep, this is something I plan to get to eventually. The idea is to have a "profile" system where you can save the current brightness and apply it from the tray, hotkeys, and using the Time of Day Adjustments.
